cert/init.sls

```diff
@@ -43,4 +43,10 @@ cert_packages:
     - mode: {{ key_mode }}  
   {% endif %}
 
+{% if grains['os_family']=="Debian" %}
+  cmd.run:
+    - name: update-ca-certificates
+    - runas: root
```

Comment:
True I did the modification.
I don't really know saltstack well.
Thanks for your advice.